src/mdns.rs

```diff
@@ -3,64 +3,89 @@
 use crate::devices::SharedDevices;
 use log::info;
 use std::net::IpAddr;
-use std::sync::Arc;
+use std::any::Any;
+use std::sync::{Arc, Mutex as StdMut};
```

Comment:
I would prefer not use import alias's (StdMut)

Reply:
Removed alias.
commit 57b380b98e4b8552bf96a655921ad88fbd9a2268